papaparse.js

```diff
@@ -193,7 +193,7 @@
 	function CsvToJson(_input, _config)
 	{
 		_config = _config || {};
-		_config.dynamicTyping = _config.dynamicTyping || false;
+		_config.dynamicTyping = new DynamicTypingConfig(_config.dynamicTyping);
```

Comment:
In this case, `dynamicTyping` will always have `isIncluded`, since it's on the prototype. The original `dynamicTyping` config the user provided is always managed through the `DynamicTypingConfig` prototype.

As far as giving the user the ability to provide their own `isIncluded`, that's an interesting idea I didn't consider. Maybe if the user sets `dynamicTyping` as a function, then the `DynamicTypingConfig` object can use that instead. I'll see about adding that.